src/resources/views/corporation/tracking.blade.php

```diff
@@ -37,12 +39,22 @@
                 {!! img('character', $character->character_id, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
                 <span rel="id-to-name">{{ $character->character_id }}</span>
               </a>
+            </td>
+            {{--<td>
               @if(!is_null($character->ship_type_id))
                 <i class="pull-right" data-toggle="tooltip"
                    title="" data-original-title="{{ $character->type->typeName }}">
                   {!! img('type', $character->ship_type_id, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
                 </i>
               @endif
+
+            </td>--}}
+            <td data-order="{{ optional(optional(optional($character->user)->group)->main_character)->character_id }}>
+              <a href="{{ route('character.view.sheet', ['character_id' => optional(optional(optional($character->user)->group)->main_character)->character_id]) }}">
+                {!! img('character', optional(optional(optional($character->user)->group)->main_character)->character_id, 64, ['class' => 'img-circle eve-icon small-icon']) !!}
```

Comment:
The nested optionals here are :(. We need a better way to do this.

Reply:
The issue really comes with who is in seat and who isn't. Somehow you need to chain it. I was looking for alternatives, but failed.

Reply:
@leonjza i removed all the optional dependencies and added the ability to asyncronously resolve character_id's for their main_character in one line: 
`<div class="character-id-to-main-character">{{ $character->character_id }}</div>`

see this commit: https://github.com/eveseat/web/pull/194/commits/6d1ac5bdcfa3ad3c0d145133c21af3b147bd161e